### PR TITLE
Add user invitation events #6

### PR DESCRIPTION
**Pull request by Jarkko-10Duke**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -105,6 +105,45 @@ Events related to user management.
...
+| eventTime | Long | Milliseconds since 1970-01-01T00:00:00Z (UTC) |
+
+#### UserInvitationSent
+
+*User invitation has been sent to an invitee.*
+
+| Field name | Type | Description |
+| ---        | ---  | ---         |
+| invitationId | String | Invitation id |
+| userId | String | User id |
```
Comment (jrauv) [inline README.md]: Is this id of inviter? If so, I think it should be emphasized

File: README.md [hunk 2]
```diff
@@ -105,6 +105,45 @@ Events related to user management.
...
+| eventTime | Long | Milliseconds since 1970-01-01T00:00:00Z (UTC) |
+
+#### UserInvitationTokenGenerated
+
+*A new user invitation token has been generated. Typically this token has been delivered to an invitee by some means.*
+
+| Field name | Type | Description |
+| ---        | ---  | ---         |
+| invitationId | String | Invitation id |
+| userId | String | User id |
```
Comment (jrauv) [inline README.md]: Is this id of the user who generated the invitation token? If so, I think it should be emphasized

File: README.md (hunk 1 above)
Comment (Jarkko-10Duke) [reply README.md]: This field is not used, fixed documentation

File: README.md (hunk 2 above)
Comment (Jarkko-10Duke) [reply README.md]: This field is not used, fixed documentation

**Pull request merged**